PollBuddy-Server/frontend/src/pages/AccountInfo/AccountInfo.js

```diff
@@ -1,40 +1,86 @@
 import React, { Component } from "react";
 import "mdbreact/dist/css/mdb.css";
-import { MDBContainer } from "mdbreact";
+import { MDBContainer, MDBRow, MDBCol } from "mdbreact";
 import {Link} from "react-router-dom";
 
 
 export default class AccountInfo extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {changePassword: false};
+    this.changePassword = this.handleToggleClick.bind(this);
+  }
+
   componentDidMount(){
     this.props.updateTitle("Account Info");
   }
 
+  
+  handleToggleClick() {
+    this.setState(state => ({
+      changePassword: !state.changePassword
+    }));
+  }
+
   render() {
     return (
       <MDBContainer className="page">
         <MDBContainer className="box">
           <h1>Account Settings</h1>
-          <MDBContainer className="form-group">
-            <label htmlFor="firstnameText">First Name:</label>
-            <input placeholder="SIS" className="form-control textBox" id="firstnameText"/>
-            <label htmlFor="lastnameText">Last Name:</label>
-            <input placeholder="Man" className="form-control textBox" id="lastnameText"/>
-            <label htmlFor="usernametext">Username:</label>
-            <input placeholder="mans" className="form-control textBox" id="usernametext" readOnly/>
-            <label htmlFor="emailText">Email:</label>
-            <input placeholder="sisman@rpi.edu" className="form-control textBox" id="emailText"/>
+          <MDBContainer>
+            <MDBRow className="accountInputs">
+              <MDBCol md="6">
+                <label htmlFor="firstnameText">First Name:</label>
+                <input placeholder="SIS" className="form-control textBox" id="firstnameText"/>
+              </MDBCol>
+              <MDBCol md="6">
+                <label htmlFor="lastnameText">Last Name:</label>
+                <input placeholder="Man" className="form-control textBox" id="lastnameText"/>
+              </MDBCol>
+            </MDBRow>
+
+            <MDBRow className="accountInputs">
+              <MDBCol md="6">
+                <label htmlFor="usernametext">Username:</label>
+                <input value="mans" className="form-control textBox" id="usernametext" readOnly disabled/>
+              </MDBCol>
+              <MDBCol md="6">
+                <label htmlFor="emailText">Email:</label>
+                <input placeholder="sisman@rpi.edu" className="form-control textBox" id="emailText"/>
+              </MDBCol>
+            </MDBRow>
+
+            <MDBRow className="accountInputs">
+              <MDBCol md="6">
+                <label htmlFor="institution">Institution:</label>
+                <input placeholder="Rensselaer Polytechnic Institute" className="form-control textBox" id="institution"/>
```

Comment:
I'm not sure how we want to handle the institution box yet. I think eventually we'll want to make it have some mechanism to disconnect/connect to a school, but since we don't a readOnly textbox seems reasonable for now.